README.md

```diff
@@ -23,13 +36,9 @@ For Mac ver. 10.15.6 or greater:
 
 7. Drag `cron` from finder to the list of apps in Full Disk Access. Ensure `cron` is checked prior to closing the window. 
 
-### Quick Start
-1. On your Terminal, navigate to the smart-files repo
-2. Run `poetry install`
-3. Run `poetry run smart-files` to display options
 
 ### Add Scheduled Sorting Job
```

Comment:
Where did the quick start section go? I dont see anything in here saying "in order to use smart-files dowlaod it using pip/pip3" with a code box showing an example. Can we make that happen?

Reply:
Just updated it in the IMPORTANT block